int16 joint_1
int16 joint_2
int16 joint_3
int16 joint_4
int16 joint_5
int16 joint_6
int16 joint_5post
int16 joint_6post
int16 gripperPos
